network-api/networkapi/wagtailpages/tests/buyersguide/test_homepage.py

```diff
@@ -261,16 +262,16 @@ def test_get_editorial_content_index_no_such_page(self):
         self.assertEqual(result, None)
 
     def test_bg_home_page_with_cta(self):
-        url = self.bg.url
-        response = self.client.get(url)
+        self.bg.call_to_action = buyersguide_factories.BuyersGuideCallToActionFactory()
+        self.bg.save()
+        response = self.client.get(self.bg.url)
         self.assertEqual(response.status_code, 200)
         self.assertIsNotNone(response.context['featured_cta'])
```

Comment:
What do you think about saving the created call to action and checking that the one in the context is really the one we expect? Any value other than `None` could pass this test.

Reply:
Hey Tibor, great call! Updated 😄